src/index.js

```diff
@@ -14,10 +14,38 @@ var ajaxView = require('./views/ajax');
 var dataView = require('./views/data');
 var logsView = require('./views/logs');
 
-function render(initPromise) {
+const state = {
+    expanded: localGet('expanded', false)
+};
+
+var LOCAL_STORAGE_KEY = 'glimpse-hud';
+
+function localGet(key, defaultValue) {
+    try {
+        var localState = JSON.parse(localStorage.getItem(LOCAL_STORAGE_KEY));
+
+        return localState[key] === undefined
+            ? defaultValue
+            : localState[key];
+    } catch (e) {
+        return defaultValue;
+    }
+}
+
+function localSet(key, value) {
+    state[key] = value;
+
+    try {
+        localStorage.setItem(LOCAL_STORAGE_KEY, JSON.parse(state));
+    } catch (e) {
+        // do nothing
+    }
+}
```

Comment:
Can we shift this into https://github.com/Glimpse/Glimpse.Client.Hud/blob/dev/src/lib/util.js or create https://github.com/Glimpse/Glimpse.Client.Hud/blob/dev/src/lib/store.js.